README.md

```diff
@@ -223,6 +223,8 @@ end
 
   - [swift-snapshot-testing-stitch](https://github.com/Sherlouk/swift-snapshot-testing-stitch/) adds the ability to stitch multiple UIView's or UIViewController's together in a single test.
 
+  - [SnapshotTestingHEIC](https://github.com/alexey1312/SnapshotTestingHEIC) adds HEIC format support.
```

Comment:
Is it worth highlighting the benefits of this? Potentially something like:

```suggestion
  - [SnapshotTestingHEIC](https://github.com/alexey1312/SnapshotTestingHEIC) adds image support using the HEIC storage format which reduces file sizes in comparison to PNG.
```

Reply:
Np, I'll fix it 👌